bootstrap/12-pm.sh

```diff
@@ -50,6 +50,14 @@ elif is_debian_based; then
     add_package_to_includes inetutils-ping
 fi
 
+if ${ENABLE_WIRELESS}; then
+    if is_alpine; then
+        add_package_to_includes wpa_supplicant
+    elif is_debian_based; then
+        add_package_to_includes wpasupplicant
```

Comment:
Let's test on Devuan and Debian

Reply:
`is_debian_based` means Devuan, Debian, Kali and Ubuntu.